src/rgw/rgw_sal_daos.h

```diff
@@ -185,6 +185,11 @@ class DaosBucket : public Bucket {
 
   DaosBucket(DaosStore* _st) : store(_st), acls() {}
 
+  DaosBucket(const DaosBucket& _daos_bucket) : store(_daos_bucket.store), acls(), coh(DAOS_HDL_INVAL), dfs(nullptr) {
+    //TODO: deep copy all objects
+    //daos_duplicate_handle(_daos_bucket.coh);
+  }
+
```

Comment:
Maybe we could get rid of the copy constructor and just zero out the `coh` and `dfs` handles inside the `clone` method? This way we don't have to handle all the cases.

Reply:
its part of the crash fix